tests/foreman/ui/test_ansible.py

```diff
@@ -602,6 +604,56 @@ def test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors(
             assert f'notice Install the {package_name} package' in notice_log['permission_denied']
             assert f'Installed: rubygem-{package_name}' in notice_log['permission_denied']
 
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
+    def test_positive_debug_log_for_host(
+        self, target_sat, module_org, module_location, module_ak_with_cv
+    ):
+        """Verify debug mode output for host
+
+        :id: 06e0a95c-530b-11ef-a28c-36dcd2c0c415
+
+        :steps:
+            1. Enable Ansible Debug
```

Comment:
I don't see any ansible debug or verbose output being set in this test

Reply:
Ansible debug mode is not enabled because it does not provide the correct output due to a bug. Therefore, i removed this line and there is no need to set the debug mode now. Ansible debug mode is currently set by default. Once the bug is fixed, I will parameterize the debug mode to view the output accordingly.